Bump Go version and refresh docs (#267)
moia-oss/aws-apigateway-exporter: README.md

  - "arn:aws:apigateway:eu-central-1::/usageplans*"
```

Replace `eu-central-1` in the policy when running the exporter with another `--region`.

Make sure your machine has Go 1.26 installed, then run `make build-linux` to build the Linux binary. To build
both Linux and Darwin amd64 binaries, run `make build`. To build the container image, run `make docker-build`.
You can also use the provided container images:

> https://gallery.ecr.aws/s6w2n1r6/aws-apigateway-exporter

Author: ```suggestion
https://gallery.ecr.aws/moia-oss/aws-apigateway-exporter
```

For running the latest provided image:

> `docker run -p 9389:9389 public.ecr.aws/s6w2n1r6/aws-apigateway-exporter:latest`

Author: ```suggestion
`docker run -p 9389:9389 public.ecr.aws/moia-oss/aws-apigateway-exporter:latest`
```

We recommend to use tagged versions in production.

> You can find the available tags on [AWS ECR Public Gallery](https://gallery.ecr.aws/s6w2n1r6/aws-apigateway-exporter).

Author: ```suggestion
You can find the available tags on [AWS ECR Public Gallery](https://gallery.ecr.aws/moia-oss/aws-apigateway-exporter).
```
